src/emitter/emitter_client.ts

```diff
@@ -76,26 +87,44 @@ export class EmitterClient {
       txOptions,
       () => undefined,
       this.contract.call(
-        'swap_backstop',
-        ...this.spec.funcArgsToScVals('swap_backstop', { new_backstop_id })
+        'queue_swap_backstop',
+        ...this.spec.funcArgsToScVals('queue_swap_backstop', contractArgs)
       )
     );
   }
 
-  async drop(
+  async cancelSwapBackstop(
     source: string,
     sign: (txXdr: string) => Promise<string>,
     network: Network,
-    txOptions: TxOptions,
-    new_backstop_id: Address | string
+    txOptions: TxOptions
+  ): Promise<ContractResult<undefined>> {
+    return await invokeOperation<undefined>(
+      source,
+      sign,
+      network,
+      txOptions,
+      () => undefined,
+      this.contract.call(
+        'cancel_swap_backstop',
+        ...this.spec.funcArgsToScVals('cancel_swap_backstop', {})
+      )
+    );
+  }
+
+  async swapBackstop(
+    source: string,
+    sign: (txXdr: string) => Promise<string>,
+    network: Network,
+    txOptions: TxOptions
   ): Promise<ContractResult<undefined>> {
     return await invokeOperation<undefined>(
       source,
       sign,
       network,
       txOptions,
       () => undefined,
-      this.contract.call('drop', ...this.spec.funcArgsToScVals('drop', { new_backstop_id }))
```

Comment:
Was the drop function intentionally removed?

Reply:
Yes, this function is not on the emitter anymore/